src/app/view/components/_common/pattern-library/pattern-library.html

```diff
@@ -45,8 +45,14 @@ <h3 class="section__title">Data Table</h3>
             </article>
             <h3 class="section__title">Action header</h3>
             <article>
-                <action-header [selected]="selectedCars.length > 0" [selectedItems]="selectedCars" [actionButtonItems]="buttonActions" [primaryCommand]="actionHeaderLog"></action-header>
-                <p-dataTable [value]="dataTableDummyData" [rows]="6" [paginator]="true" [pageLinks]="5" [(selection)]="selectedCars" dataKey="vin">
+                <action-header [selected]="selectedCars.length > 0" 
+                               [selectedItems]="selectedCars" 
+                               [actionButtonItems]="buttonActions" 
+                               [primaryCommand]="actionHeaderLog">
+
+                    <input #gf pInputText type="text" placeholder="Global Search" />
+                </action-header>
+                <p-dataTable [value]="dataTableDummyData" [rows]="6" [paginator]="true" [pageLinks]="5" [(selection)]="selectedCars" dataKey="vin" [globalFilter]="gf">
```

Comment:
Global filter wasn't working, I try to add the reference but still not working, not sure why.